src/functions/sendCandidateResults/application/templates/appeal-your-test.ts

```diff
@@ -18,9 +18,9 @@ export enum AppealUrls {
 
 export const howToAppealTemplate = (url: string, category?: TestCategory, riding?: boolean): string => {
   return  `
-# How to appeal your ${riding ? 'motorcycle' : 'driving'} test
+# How to appeal your ${category === TestCategory.ADI3 ? 'test' : category === TestCategory.SC ? 'standards check' : (riding ? 'motorcycle test' : 'driving test')} 
```

Comment:
For this you could do:

const getTestType = (category: TestCategory | undefined, riding: boolean | undefined): string => {
  switch (category) {
  case TestCategory.ADI3:
    return 'test';
  case TestCategory.SC:
    return 'standards check';
  default:
    return riding ? 'motorcycle test' : 'driving test';
  }
};

And then just use it like `# How to appeal your ${getTestType(category, riding)}`

within the returned string